pmda/util.py

```diff
@@ -211,6 +211,7 @@ def pair_wise_rmsf(mu1, mu2, t1, t2, sos1, sos2):
         given atom selection
     """
     T = t1 + t2
-    M = sos1 + sos2 + (t1 * t2/T) * (mu2 - mu1)**2
+    mean = (mu1 + mu2)/2
+    sos = sos1 + sos2 + (t1 * t2/T) * (mu2 - mu1)**2
     rms_fluctuations = np.sqrt(M.sum(axis=1)/T)
-    return rms_fluctuations
+    return mean, sos, rms_fluctuations
```

Comment:
Here the order is `mean, sumofsq, ...` whereas for `res` it is `sumofsq, mean, ...`. Better to be consistent. Personally, I would order it by moments, i.e., mean, stdev (sumofsq), ...

Reply:
The function needs tests. See above for suggestion on how to do tests.